index.njk

```diff
@@ -14,4 +14,11 @@ menu:
 
 <hr>
 
+<h2>Pinned Post:</h1>
+
+{% set pinnedpost = search.pages("'pinned'") %}
+{% include "templates/pinnedpost.njk" %}
+
+<hr>
+
```

Comment:
This is a specific feature that most blogs don't have. If there isn't any pinned post, the title shouldn't be visible. It should be wrapped in a `if`.